tests/benchmarking/test_benchmarking.py

```diff
@@ -241,7 +362,6 @@ def test_chain_found(self, large_graph, benchmark):
         )
         assert result is not None
 
-    @pytest.mark.xfail("grimp.exceptions.ModuleNotPresent")
```

Comment:
FYI @seddonym - see here that fixing the graph means that the module `mypackage.data.vendors.4053192739.6373932949` now exists!